.github/workflows/test.yml

```
@@ -61,7 +61,7 @@ jobs:
       - run: julia --project --color=yes --check-bounds=yes -e 'import Pkg; Pkg.build(); Pkg.test(; coverage=true)'
         env:
           JULIA_PKG_SERVER: ${{ matrix.pkg-server }}
-          JULIA_PKG_TEST_QUIET: "true" # "true" is the default. i.e. tests are quiet when this env var isn't set
+          JULIA_PKG_TEST_QUIET: "false" # "true" is the default. i.e. tests are quiet when this env var isn't set
```

@DilumAluthge It'd be nice to see that this works at least once before merge. So let's leave this on for now and re-run CI until we see the `@warn "Registry update attempt failed...` log, then turn it off before merge

```suggestion
          JULIA_PKG_TEST_QUIET: "true" # "true" is the default. i.e. tests are quiet when this env var isn't set
```